concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -474,7 +482,41 @@ jobs:
             passed: ['generate-bosh-config']
             trigger: true
           - get: cf-secrets
-
+          - get: metrics-cert
+          - get: vpc-tfstate
+          - get: concourse-tfstate
+          - get: bosh-tfstate
+      - task: generate-metrics-certificates
+        config:
+          platform: linux
+          image: docker:///governmentpaas/curl-ssl
+          inputs:
+            - name: metrics-cert
+              path: existing-metrics-cert
+          outputs:
+            - name: generated-metrics-cert
+          params:
+            SYSTEM_DNS_ZONE_NAME: {{system_dns_zone_name}}
+          run:
+            path: sh
+            args:
+              - -e
+              - -c
+              - |
+                tar xzvf existing-metrics-cert/metrics-cert.tar.gz
+                if [ -f metrics.crt ] && [ -f metrics.key ]; then
+                  echo Certificate and private key already created, nothing to do
+                  cp existing-metrics-cert/metrics-cert.tar.gz generated-metrics-cert/.
+                  exit 0
+                fi
+                openssl req -x509 -newkey rsa:2048 -keyout metrics.key \
+                  -out metrics.crt -days 365 -nodes -subj \
+                  "/C=UK/ST=London/L=London/O=GDS/CN=metrics.{{deploy_env}}.${SYSTEM_DNS_ZONE_NAME}"
```

Comment:
We missed this out. I changed it but probably it got overwritten by other PR.

@saliceti will fix it in other PR #167